integration/tests/cook/test_master_slave.py

```diff
@@ -1,12 +1,13 @@
 import logging
 import os
-import pytest
-import time
 import unittest
 
+import pytest
```

Comment:
Why move this import lower down?

Reply:
From https://www.python.org/dev/peps/pep-0008/#imports:

> Imports should be grouped in the following order:
>
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
>
> You should put a blank line between each group of imports.